src/Frontend/src/components/CopyToClipboard.vue

```diff
@@ -2,9 +2,16 @@
 import { Tippy, TippyComponent } from "vue-tippy";
 import { useTemplateRef } from "vue";
 
-const props = defineProps<{
-  value: string;
-}>();
+const props = defineProps({
+  value: {
+    type: String,
+    required: true,
+  },
+  isIconOnly: {
+    type: Boolean,
+    default: false, // Default value
+  },
+});
```

Comment:
default values are handled differently elsewhere in SP

```suggestion
const props = defineProps<{
    value: string,
    isIconOnly?: boolean
}().withDefaults({ isIconOnly: false })
```